src/parser/jobs/brd/Util.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * @author Yumiya
+ */
+import Module from 'parser/core/Module'
+
+export default class Util extends Module {
```

Comment:
I'm a little wary of this being a module unto itself but I get what you're doing with it. Keep in mind modules do have overhead on every event, currently.